tests/test_coords.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import pytest
+
+import spherely
+
+
+def test_get_y_lng() -> None:
+    # scalar
+    a = spherely.Point(1.5, 2.6)
```

Comment:
```suggestion
    a = spherely.point(1.5, 2.6)
```

(also for the other points created below)